_includes/meeting.html

```diff
@@ -23,7 +23,7 @@
   <ul>
     <li>
       <a href="{{ meeting.plans }}">Plans and suggested preparation</a>
-      {% assign meeting_date = meeting.date | date: '%F' %}
+      {% assign meeting_date = meeting.date | date: '%F 18:30 PM' %}
```

Comment:
I think this date is only used for the following conditional (to see if the meeting is in the future or not) so it might be enough to rely on the full ISO 8601 date generated by `%F` (e.g. `2010-01-01`).

Do we still want to include the time in that case?

Reply:
I've removed it